tools/publish_http.py

```diff
@@ -228,8 +228,8 @@ def main(argv):
         logger.info('Install your package using the following command:')
     else:
         logger.info('Install your package using the following commands:')
-        logger.info('dcos package repo add --index=0 {}-aws {}'.format(self._pkg_name, universe_url))
-    logger.info('dcos package install {}'.format(self._pkg_name))
+        logger.info('dcos package repo add --index=0 {}-aws {}'.format(package_name, universe_url))
+    logger.info('dcos package install {}'.format(package_name))
```

Comment:
It does currently log a message like this: https://github.com/mesosphere/dcos-commons/blob/master/tools/publish_aws.py#L168

Difference is `publish_aws` doesn't automatically attempt to add the universe, reasoning being that publish_aws is more often called in CI where automatically adding the universe at build-time may be undesirable (nothing says this couldnt be changed later though)